docs/Developers/Linter_Bears_Advanced.rst

```diff
@@ -41,6 +41,31 @@ The path of the temporary configuration file can be accessed inside
         def create_arguments(filename, file, config_file):
             return "--use-config", config_file
 
+The ``generate_config()`` should not return ``config_file`` if the user wants
+to use his own tool-specific config, to increase performance. If a config
+file is supplied by user that means he won't need the setting provided by
+coala to generate the config file. Therefore, in such scenarios, it is advised
+to return ``None`` in ``generate_config()``.
+For more information on coala settings:-
+http://api.coala.io/en/latest/Developers/coala_settings.html
```

Comment:
That reference doesn't really belong here^^

Reply:
I thought it might be helpful if the reader doesn't know about the settings in Coala. But I will remove it if you say so 😄

Reply:
The reader should, as it's developer's documentation ;)

PS:
coala is always written with a lowercase c ;)